.github/workflows/netpol-e2e-test.yaml

```diff
@@ -230,10 +237,133 @@ jobs:
+          # wait for webhook to be ready
+          POD_IP=`kubectl get pod -l app=intents-operator -n otterize-system -o=jsonpath='{.items[0].status.podIP}'`
+          kubectl wait -n otterize-system --for=jsonpath='{.subsets[0].addresses[0].ip}'=$POD_IP endpoints/intents-operator-webhook-service
+
+
+      - name: Wait for Tutorial services
+        run: |-
+          kubectl wait pods -n otterize-tutorial-npol -l app=client --for condition=Ready --timeout=180s
+          kubectl wait pods -n otterize-tutorial-npol -l app=client-other --for condition=Ready --timeout=180s
+          kubectl wait pods -n otterize-tutorial-npol -l app=server --for condition=Ready --timeout=180s
+
+      - name: Before apply intents
+        run: |-
+          CLI1_POD=`kubectl get pod --selector app=client -n otterize-tutorial-npol -o json | jq -r ".items[0].metadata.name"`
+          CLI2_POD=`kubectl get pod --selector app=client-other -n otterize-tutorial-npol -o json | jq -r ".items[0].metadata.name"`
+          echo Client: $CLI1_POD      client_other: $CLI2_POD
+          source .github/workflows/test-bashrc.sh
+          
+          # using 14 because the log repeat itself every 14 lines
+          echo check client log
+          wait_for_log $CLI1_POD 10 "Hi, I am the server, you called, may I help you?"
+          
+          echo check client other log
+          wait_for_log $CLI2_POD 10 "Hi, I am the server, you called, may I help you?"
+
+      - name: Apply intents and test connectivity
+        run: |-
+          CLI1_POD=`kubectl get pod --selector app=client -n otterize-tutorial-npol -o json | jq -r ".items[0].metadata.name"`
+          CLI2_POD=`kubectl get pod --selector app=client-other -n otterize-tutorial-npol -o json | jq -r ".items[0].metadata.name"`          
+          echo Client: $CLI1_POD      client_other: $CLI2_POD
+          source .github/workflows/test-bashrc.sh
+          
+          kubectl apply -f https://docs.otterize.com/code-examples/automate-network-policies/intents.yaml
+          
+          # should not work at first because there is no allow DNS netpol
+          echo check client log
+          wait_for_log $CLI1_POD 10 "curl timed out"
+          
+          # should be blocked (using 3 because the log should repeat itself every 3 lines)
+          echo check client other log
+
+          wait_for_log $CLI2_POD 10 "curl timed out"
+          
+          kubectl apply -f .github/workflows/allowDNS.yaml -n otterize-tutorial-npol
+          
+           # should work because there is an applied intent and allowDNS netpol
+          echo check client log
+          wait_for_log $CLI1_POD 10 "Hi, I am the server, you called, may I help you?"
+          
+           # should be blocked (using 3 because the log should repeat itself every 3 lines)
+          echo check client other log
+
+          wait_for_log $CLI2_POD 10 "curl timed out"
+  
+  
```

Comment:
This test is actually checking nothing is totally broken when egress enabled, but I suggest there will be another service, `server2` that only `client` will call (won't be called by `client-other`). When intents applied we should expect communication with `server2` to stop, and would assert that egress policy actually work with blocking the *client* un-declared calls to servers